frontend/lib/verification/verification_encoder.dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:convert';
+
+import 'package:fixnum/fixnum.dart';
+
+import '../identification/base_card_details.dart';
+import '../identification/protobuf/card_verify_model.pb.dart';
+import 'verification_card_details.dart';
+
+String encodeVerificationCardDetails(
+    VerificationCardDetails verificationCardDetails) {
+  final cardDetails = verificationCardDetails.cardDetails;
+
+  final cardVerifyModel = CardVerifyModel();
+
+  cardVerifyModel.fullName = cardDetails.fullName;
+  cardVerifyModel.randomBytes = List<int>.from(cardDetails.randomBytes);
+  cardVerifyModel.expirationDate = Int64(cardDetails.unixExpirationDate);
```

Comment:
What does this `Int64` method do?

Reply:
Converts it to protobufs internal int64 datatype. I think it didn't want to accept a normal int for some reason